components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -334,6 +334,9 @@ RewardsServiceImpl::RewardsServiceImpl(Profile* profile)
   content::URLDataSource::Add(profile_,
                               std::make_unique<BraveRewardsSource>(profile_));
   ready_ = std::make_unique<base::OneShotEvent>();
+
+  if (base::FeatureList::IsEnabled(features::kVerboseLoggingFeature))
+    persist_log_level_ = kDiagnosticLogMaxVerboseLevel;
```

Comment:
Maybe worth checking the performance impact on Android?

Reply:
cc @deeppandya

Reply:
I don't think it would have any impact on performance. I can run through the branch and quickly check.